addons/website_sale/models/res_config_settings.py

```diff
@@ -1,40 +1,76 @@
+        group='base.group_portal,base.group_user,base.group_public',
+    )
 
+    # Modules
     module_account = fields.Boolean("Invoicing")
-    module_website_sale_picking = fields.Boolean('On Site Payments & Picking')
+    module_delivery_mondialrelay = fields.Boolean("Mondial Relay Connector")
+    module_website_sale_autocomplete = fields.Boolean("Address Autocomplete")
+    module_website_sale_comparison = fields.Boolean("Product Comparison Tool")
+    module_website_sale_picking = fields.Boolean("On Site Payments & Picking")
+    module_website_sale_wishlist = fields.Boolean("Wishlists")
 
-    cart_recovery_mail_template = fields.Many2one('mail.template', string='Cart Recovery Email', domain="[('model', '=', 'sale.order')]",
-                                                  related='website_id.cart_recovery_mail_template_id', readonly=False)
-    cart_abandoned_delay = fields.Float(string="Send After", related='website_id.cart_abandoned_delay', readonly=False)
-    send_abandoned_cart_email = fields.Boolean('Abandoned Email', related='website_id.send_abandoned_cart_email', readonly=False)
+    # Website-dependent settings
     add_to_cart_action = fields.Selection(related='website_id.add_to_cart_action', readonly=False)
+    cart_recovery_mail_template = fields.Many2one(
+        related='website_id.cart_recovery_mail_template_id',
+        readonly=False,
+    )
+    cart_abandoned_delay = fields.Float(
+        related='website_id.cart_abandoned_delay',
+        readonly=False,
+    )
+    send_abandoned_cart_email = fields.Boolean(
+        string="Abandoned Email",
+        related='website_id.send_abandoned_cart_email',
+        readonly=False,
+    )
+    salesperson_id = fields.Many2one(
+        related='website_id.salesperson_id',
+        readonly=False,
+    )
+    salesteam_id = fields.Many2one(related='website_id.salesteam_id', readonly=False)
+    website_sale_prevent_zero_price_sale = fields.Boolean(
+        string="Prevent Sale of Zero Priced Product",
+        related='website_id.prevent_zero_price_sale',
+        readonly=False,
+    )
+    website_sale_contact_us_button_url = fields.Char(
+        related='website_id.contact_us_button_url',
+        readonly=False,
+    )
+    website_sale_enabled_portal_reorder_button = fields.Boolean(
+        related='website_id.enabled_portal_reorder_button',
+        readonly=False,
+    )
+    show_line_subtotals_tax_selection = fields.Selection(
+        related='website_id.show_line_subtotals_tax_selection',
+        readonly=False,
+    )
 
-    module_delivery_mondialrelay = fields.Boolean("Mondial Relay Connector")
-    group_product_pricelist = fields.Boolean(
-        compute='_compute_group_product_pricelist', store=True, readonly=False)
-
-    enabled_extra_checkout_step = fields.Boolean(string="Extra Step During Checkout")
-    enabled_buy_now_button = fields.Boolean(string="Buy Now")
+    # Additional settings
 
```

Comment:
Line or not line after the comment announcing the group of fields? That's the question ^^